pkg/infrastructure/platform/platform.go

```diff
@@ -51,12 +50,9 @@ import (
 func ProviderForPlatform(platform string, fg featuregates.FeatureGate) (infrastructure.Provider, error) {
 	switch platform {
 	case awstypes.Name:
-		if types.ClusterAPIFeatureGateEnabled(platform, fg) {
+		if types.ClusterAPIFeatureGateEnabled(platform, fg) || fg.Enabled(features.FeatureGateInstallAlternateInfrastructureAWS) {
```

Comment:
Can `features.FeatureGateInstallAlternateInfrastructureAWS` be removed now from the installer ?

Reply:
Yes. We can do that one before this PR, but it'll involve changes to `openshift/api` to remove the gate.

Reply:
Actually, if we just ignore the gate in the Installer, then later on we can remove it from `openshift/api` without having to bump the api version in the installer.

Reply:
I'm ok either way, but lean toward removing the `FeatureGateInstallAlternateInfrastructureAWS` feature gate.